cmd/root.go

```diff
@@ -32,15 +32,17 @@ var (
 		metricsAddress         string
 		otel                   bool
 		otelAddress            string
+		emulatedEtcdVersion    string
```

Comment:
Re-opening the discussion on the emulatedEtcdVersion flag here @berkayoz, @bschimke95 and @marco6.

It would not make sense for users to drop the version, we may only bump it in the future if some new fancy feature requires it. 

If a user wishes to disable features they should do so in the api-server.
As a consequence, we do not have to make this `emulatedEtcdVersion` configurable.